cumulus/ansible/tasks/playbooks/roles/ec2-pod/tasks/launch.yml

```diff
@@ -124,26 +124,34 @@
     instance_tags:
       Name: "ec2_pod_{{ cluster_id }}_node"
       ec2_pod: "{{ cluster_id }}"
-      ec2_pod_instance_name: node
+      ec2_pod_instance_name: data
     count_tag:
       ec2_pod: "{{ cluster_id }}"
-      ec2_pod_instance_name: node
+      ec2_pod_instance_name: data
     exact_count: "{{ node_instance_count }}"
```

Comment:
gridengine/site.yml expects the nodes to be in the data group. Group membership is determined (by the cumulus.py dynamic inventory) from the ec2_pod_instance_name tag.